api/representation.go

```diff
@@ -41,9 +41,9 @@ func (tr taskRequest) ToTaskConfig() (config.TaskConfig, error) {
 			if tr.SourceInput.Services.Names != nil {
 				si.Names = *tr.SourceInput.Services.Names
 			}
-			tc.SourceInput = si
+			tc.ModuleInput = si
```

Comment:
consider renaming instances of `si` variable to `mi` or other name that reflects `ModuleInput`

also suggest doing this for https://github.com/hashicorp/consul-terraform-sync/pull/592

Reply:
I can make this change in the backend PR since this is out-of-scope for this PR